_includes/IDR_API_example_script.md

```diff
@@ -162,54 +162,47 @@ for a in session.get(url).json()['annotations']:
 ----
 
 
-## Attributes (e.g. Gene, Phenotype...)
+## Attributes (e.g. Gene Symbol, Phenotype Term Accession...)
 
-### Get Screens that are annotated with gene:
+### Load all the possible values associated to a specific key e.g. Gene Symbol:
 ```python
-SCREENS_PROJECTS_URL = "https://idr.openmicroscopy.org/mapr/api/{key}/?value={value}"
-
-qs = {'key': 'gene', 'value': 'CDC20'}
-url = SCREENS_PROJECTS_URL.format(**qs)
-for s in session.get(url).json()['screens']:
-    screen_id = s['id']
-    print (s['id'], s['name'])
-```
-
-----
-
-
-### Get Plates in Screen that are annotated with gene:
-```python
-PLATES_URL = "https://idr.openmicroscopy.org/mapr/api/{key}/plates/?value={value}&id={screen_id}"
-
-qs = {'key': 'gene', 'value': 'CDC20', 'screen_id': screen_id}
-url = PLATES_URL.format(**qs)
-for p in session.get(url).json()['plates']:
-    plate_id = p['id']
-    print (p['id'], p['name'])
+KEYS_SEARCH = "https://idr.openmicroscopy.org/searchengine/api/v1/resources/{type}/searchvaluesusingkey/?key={key}"
+
+values = []
+qs1 = {'type': 'image', 'key': KEY}
+url = KEYS_SEARCH.format(**qs1)  
+json = session.get(url).json()
```

Comment:
You could just use `json = requests.get(url).json()` here, since you don't need a session.
And below you could do `values = [d["Value"] for d in json["data"]` if you wanted to save some space (maybe less readable)?

Reply:
I want to keep as readable as possible and avoid empty values